ruby-training/trainees/david/test_katas/01/spec/calculator_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rspec/autorun'
+require_relative '../calculator'
+
+describe Calculator do
+    describe "#add" do
```

Comment:
This is the notation for an instance method ("#"). Instance methods are the ones you call on class instance objects like so:

```ruby
calculator_instance.add(a, b)
```

You wrote your methods as class methods:
```ruby
Calculator.add(a, b)
```

The notation for class method descriptions in rspec is: 

```ruby
describe ".add" do
  .
  . 
  .
end
```
Using the dot (.) notation.